### PR TITLE
Make "Assessment or further help" sections consistent

### PR DESCRIPTION
**Pull request by glmvc**
#### Summary

Like discussed and suggested in pr #15579:

- adapt the "Assessment or further help" sections (at the end of assessments and test your skill articles) to be more consistent across the HTML and CSS learning modules
- the other modules will follow if i move on in the learning section and if wanted of course

#### Motivation

To make things more consistent...

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/html/tables/structuring_planet_data/index.md [hunk 1]
```diff
@@ -69,13 +69,13 @@ You can also [see the example live here](https://mdn.github.io/learning-area/htm
...
 
-If you would like your work assessed, or are stuck and want to ask for help:
+If you would like your work assessed or are stuck and want to ask for help:
 
 1. Put your work into an online shareable editor such as [CodePen](https://codepen.io/), [jsFiddle](https://jsfiddle.net/), or [Glitch](https://glitch.com/).
 2. Write a post asking for assessment and/or help at the [MDN Discourse forum Learning category](https://discourse.mozilla.org/c/mdn/learn/250). Your post should include:
 
     - A descriptive title such as "Assessment wanted for Structuring planet data".
-    - Details of what you have already tried, and what you would like us to do, e.g. if you are stuck and need help, or want an assessment.
+    - Details of what you have already tried and what you would like us to do if you're stuck and need help or want an assessment.
```
Comment (estelle) [inline files/en-us/learn/html/tables/structuring_planet_data/index.md:78]: while the comma should be gone, the eg should be there. If the reason to reach out is limited only to "being stuck and needing help" or "needing an assessment." and not to ask a clarifying questions, then the change is ok, but I think we want to leave it open, so the original with e.g., i.e., or for example is better

**Review by estelle — CHANGES_REQUESTED** [review]
Thank you. 
Most of my edits are all the same: based on removing the e.g. My reasoning is on the last one.

File: files/en-us/learn/html/tables/structuring_planet_data/index.md (hunk 1 above)
Comment (glmvc) [reply files/en-us/learn/html/tables/structuring_planet_data/index.md:78]: I agree 👍 this change was suggested by an other contributor in a separate pr that' s why I thought it was wanted/requested this way...

But anyway, I will go with your suggestion:

...you would like us to do **; i.e. what** you **are** stuck **on** and need help **with or if you** want an assessment.

that's how it makes the most sense and sounds and reads best in my opinion 🥳

**Comment by dipikabh** [comment]
About the sentence:

> "Details of what you have already tried, and what you would like us to do, e.g. if you are stuck and need help, or want an assessment."

I agree with @estelle that the two commas should go and that we should state the latter phrase as an example. My only nit is about how we state that example.

As per MDN's writing style guide, we should use contractions such as [i.e. and e.g. inside parenthesis](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Writing_style_guide#latin_abbreviations). I'm not sure how widely this guideline has been followed on MDN, but if we try to stick to this guideline in this instance, we can rephrase the sentence as:

> "Details of what you have already tried and what you would like us to do; for example, tell us if you're stuck and need help or want an assessment."

**Comment by glmvc** [comment]
Hi @dipikabh, thanks for your input and suggestion, I think we should go with the sentence you suggested 🎉
Should we wait for @estelle's thoughts or can I start replacing the sentences?
Thank you in advance :)

**Comment by estelle** [comment]
Anything @dipikabh suggests is fine by me. She's an excellent tech editor!

**Comment by glmvc** [comment]
hey @dipikabh, I've changed the sentences according to your suggestion (HTML and CSS Learning Areas).

**Review by dipikabh — APPROVED** [review]
👍

**Comment by glmvc** [comment]
@estelle i think we are ready to merge :)

**Review by estelle — APPROVED** [review]
thank you!

**Pull request merged**